ios-swiftui/realm-template-apps-swiftui/Views/NewTodo.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import RealmSwift
+
+/// Create a new task.
+struct NewTodo: View {
+    // Appending objects to the ObservedResults collection
+    // adds them to the realm.
+    @ObservedResults(Todo.self) var todos
+    @State private var newTodo = Todo()
```

Comment:
Could these members have comments to explain their purpose? Or a more descriptive variable name?